src/resources/postcss/tribe-events-full.pcss

```diff
@@ -2199,8 +2208,9 @@ input[name*='tribe-bar-']:-moz-placeholder {
 ========================================================================== */
 .tribe-theme-twentyfifteen,
 .tribe-theme-parent-twentyfifteen {
+	--tec-color-background-events: var(--tec-color-background);
```

Comment:
where does `--tec-color-background` come from? also, are we updating all the old css files?